### PR TITLE
Feature/layer etc

### PR DESCRIPTION
**Pull request by petersteneteg**
Fixes #...

Changes proposed in this PR:
 * ...
 * ...
 * ...

This has been tested on: ...

> Thank you for contributing to the Inviwo repository! Please ensure the following: 
> - Code is error and warning free
> - Code follows the Inviwo coding guidelines: 
>   https://github.com/inviwo/inviwo/wiki/Coding-Conventions
> - New Code / Processors follows the Inviwo documentation guidelines: 
>   https://github.com/inviwo/inviwo/wiki/Documentation-guide
> - New Code / processors are tested in a unittest and / or regression test: 
>   https://github.com/inviwo/inviwo/wiki/Manual#testing
> 
> Please remember to specify the environment this code been compiled and tested on:
> Operating System / Compiler / IDE / QT / CMake / Python versions.

### CODE REVIEW

File: modules/base/src/processors/layersequencesource.cpp [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+namespace inviwo {
+
+// The Class Identifier has to be globally unique. Use a reverse DNS naming scheme
+const ProcessorInfo LayerSequenceSource::processorInfo_{
+    "org.inviwo.LayerSequenceSource",  // Class identifier
+    "Layer Sequence Source",           // Display name
+    "Undefined",                       // Category
+    CodeState::Experimental,           // Code state
+    Tags::None,                        // Tags
+    R"(<Explanation of how to use the processor.>)"_unindentHelp};
```
Comment (martinfalk) [inline modules/base/src/processors/layersequencesource.cpp]: fill in!

File: modules/base/src/processors/layerseriessource.cpp [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+// The Class Identifier has to be globally unique. Use a reverse DNS naming scheme
+const ProcessorInfo LayerSeriesSource::processorInfo_{
+    "org.inviwo.LayerSeriesSource",  // Class identifier
+    "Layer Series Source",           // Display name
+    "Undefined",                     // Category
+    CodeState::Experimental,         // Code state
+    Tags::None,                      // Tags
+    R"(<Explanation of how to use the processor.>)"_unindentHelp};
+
```
Comment (martinfalk) [inline modules/base/src/processors/layerseriessource.cpp]: fill in!

File: modules/dataframe/src/processors/dataframetovector.cpp [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+namespace inviwo {
+
+// The Class Identifier has to be globally unique. Use a reverse DNS naming scheme
+const ProcessorInfo DataFrameToVector::processorInfo_{
+    "org.inviwo.DataFrameToVector",  // Class identifier
+    "Data Frame To Vector",          // Display name
+    "Undefined",                     // Category
+    CodeState::Experimental,         // Code state
+    Tags::None,                      // Tags
+    R"(<Explanation of how to use the processor.>)"_unindentHelp};
```
Comment (martinfalk) [inline modules/dataframe/src/processors/dataframetovector.cpp]: fill in!

Maybe similar to DataFrameJoin?
```
    "DataFrame",                 // Category
    CodeState::Experimental,     // Code state
    "CPU, DataFrame",            // Tags
```

**Review by martinfalk — CHANGES_REQUESTED** [review]

File: modules/dataframe/src/processors/dataframetovector.cpp [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    , floatOutport_{"floatOutport", "<description of the generated outport data>"_help}
+    , stringOutport_{"stringOutport", "<description of the generated outport data>"_help}
+    , selectedColumn_("selectedColumn", "Selected Column", dataFrame_) {
+
+    addPorts(dataFrame_, uintOutport_, floatOutport_, stringOutport_);
+    addProperties(selectedColumn_);
+}
+
+void DataFrameToVector::process() {
+    const auto col = dataFrame_.getData()->getColumn(selectedColumn_.getSelectedValue());
```
Comment (martinfalk) [inline modules/dataframe/src/processors/dataframetovector.cpp:60]: What happens to the data set previously on the other ports? That data still persists after selecting another column with a different format.

Comment (petersteneteg) [reply modules/dataframe/src/processors/dataframetovector.cpp:60]: true, didn't consider that. But they should be cleared

**Review by martinfalk — APPROVED** [review]

**Pull request merged**